lib/artist.rb

```diff
@@ -0,0 +1,15 @@
+class Artist
+  attr_reader :id,
+              :name,
+              :born,
+              :died,
+              :country
+
+  def initialize(data)
+    @id = data[:id] || data["id"]
+    @name = data[:name]|| data["name"]
+    @born = data[:born]|| data["born"]
+    @died = data[:died]|| data["died"]
+    @country = data[:country]|| data["country"]
+  end
```

Comment:
neat extra step; generally we won't let users (or other developers) pass data of either string and symbol, but force them to use one only; it makes our lives easier.